vagrant/vagrant-test.sh

```diff
@@ -103,7 +107,7 @@ done
 ## Other integration tests ##
 TEST_LIST=(
     "test/test-exec-deserialization.py"
-    "test/test-network/systemd-networkd-tests.py"
+#    "test/test-network/systemd-networkd-tests.py"
```

Comment:
Unsure why this one needed to be commented out... I'm guessing that's also temporary, right?

Reply:
And this one too. I just wanted to keep the resources busy for as short period of time as possible, as the nested KVM stuff is related only to the integration tests.